skrub/_similarity_encoder.py

```diff
@@ -202,30 +147,26 @@ class SimilarityEncoder(OneHotEncoder):
     2. To avoid dealing with high-dimensional encodings when `k` is high,
        we can use ``d << k`` prototypes ``[p1, ..., pd]`` with which
        similarities will be computed:  ``xi -> [sim(xi, p1), ..., sim(xi, pd)]``.
-       These prototypes can be automatically sampled from the input data
-       (most frequent categories, KMeans) or provided by the user.
+       These prototypes can be provided by the user. Otherwise, we recommend
+       using the MinHashEncoder or GapEncoder when taking all unique entries
+       leads to too many prototypes.
+
+    The similarity measure is based on the proportion of common n-grams between
+    two strings, but other string similarity measures could eventually be used.
```

Comment:
```suggestion
    two strings.
```

Let's not point to things that we are not doing, and that are not on our short-term todo :)